nominal/_utils.py

```diff
@@ -48,6 +48,20 @@ class CustomTimestampFormat:
     CustomTimestampFormat,
 ]
 
+Priority = Literal["P0", "P1", "P2", "P3", "P4"]
+    
+def _priority_to_conjure_priority(priority: Priority) -> scout_checks_api.Priority:
+    if priority == "P0":
+        return scout_checks_api.Priority.P0
+    elif priority == "P1":
+        return scout_checks_api.Priority.P1
+    elif priority == "P2":
+        return scout_checks_api.Priority.P2
+    elif priority == "P3":
+        return scout_checks_api.Priority.P3
+    elif priority == "P4":
+        return scout_checks_api.Priority.P4
+
```

Comment:
Should throw here for unexpected values

Reply:
I get a `Pylance error` that the code is unreachable in VSCode but I'll just try keeping it for now